tests/e2e-tests/utils/components.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+import { StoreOwnerFlow } from "./flows";
+import { clickTab, uiUnblocked } from "./index";
+
+const verifyAndPublish = async () => {
+	// Wait for auto save
+	await page.waitFor( 2000 );
+
+	// Publish product
+	await expect( page ).toClick( '#publish' );
+	await page.waitForSelector( '.updated.notice' );
+
+	// Verify
+	await expect( page ).toMatchElement( '.updated.notice', { text: 'Product published.' } );
+};
+
+/**
+ * Create simple product.
+ */
+const createSimpleProduct = async () => {
```

Comment:
Thinking ahead to other extensions etc re-using these core components, I imagine we'll want to be able to define the properties of a product when it's being created (title, price). And maybe return some kind of reference (like post-id) after creation?

Reply:
Good call, thanks! I am not returning post-id for both simple and variable products once they are created.